src/hacbs/components/PipelineRunListView/pipelinerun-actions.tsx

```diff
@@ -0,0 +1,31 @@
+import { k8sPatchResource } from '@openshift/dynamic-plugin-sdk-utils';
+import { Action } from '../../../shared/components/action-menu/types';
+import { PipelineRunModel } from '../../models/pipelineruns';
+import { PipelineRunKind } from '../../types';
+
+export const usePipelinerunActions = (pipelineRun: PipelineRunKind): Action[] => {
+  const pipelineRunStop = () => {
+    k8sPatchResource({
+      model: PipelineRunModel,
+      queryOptions: {
+        name: pipelineRun.metadata.name,
+        ns: pipelineRun.metadata.namespace,
+      },
+      patches: [
+        {
+          op: 'replace',
+          path: `/spec/status`,
+          value: 'PipelineRunCancelled',
+        },
+      ],
+    });
+  };
+
+  return [
+    {
+      cta: () => pipelineRunStop(),
```

Comment:
You're going to want to disable this action for any pipeline that isn't currently running. 
See https://github.com/openshift/console/blob/master/frontend/packages/pipelines-plugin/src/utils/pipeline-actions.tsx#L180-L211

Reply:
changed it so it is only enabled when the PLR is running, thanks